utils/verifications.py

```diff
@@ -7,99 +7,7 @@
 from tqdm import tqdm
 
 from utils.CreateDataset import MetaSegmentationDataset
-from utils.geoutils import lst_ids
-from utils.utils import get_key_recursive
-
-
-def is_valid_geom(geom):
```

Comment:
was this funtion moved somewhere else?

Reply:
is_valid_geom was already part of rasterio, but had a bug. So I had copied the function to GDL and fixed the bug temporarily while rasterio took care of that. Now the bug is solved. Rasterio has been update on cavum/velum and HPC, so we shouldn't encounter this bug again.